README.rst

```diff
@@ -178,6 +178,9 @@ Past contributors
 .. |hgugm| image:: https://github.com/pymedphys/pymedphys/raw/3f8d82fc3b53eb636a75336477734e39fa406110/docs/logos/HGUGM_200x200.png
     :target: `Hospital General Universitario Gregorio Marañón`_
 
+.. |sjs| image:: https://github.com/pymedphys/pymedphys/raw/24701eceefa0daa2ecbc065db42f9ecb901dcfb9/logos/SwerdloffCoatOfArms_200x200.png
```

Comment:
Actually, given this done via a social fork, that commit hash doesn't actually exist within the pymedphys repo yet. To make it appear, need the following:

```suggestion
.. |sjs| image:: https://github.com/sjswerdloff/pymedphys/raw/24701eceefa0daa2ecbc065db42f9ecb901dcfb9/logos/SwerdloffCoatOfArms_200x200.png
```